core/src/sigverify.rs

```diff
@@ -342,6 +387,60 @@ mod tests {
         assert_eq!(sig_len, 1);
     }
 
+    #[test]
+    fn test_untrustworthy_sigs() {
+        let required_num_sigs = 14 as u32;
+        let actual_num_sigs = 5 as u32;
+
+        let message = Message {
+            header: MessageHeader {
+                num_required_signatures: required_num_sigs as u8,
+                num_credit_only_signed_accounts: 12,
+                num_credit_only_unsigned_accounts: 11,
+            },
+            account_keys: vec![],
+            recent_blockhash: Hash::default(),
+            instructions: vec![],
+        };
+        let mut tx = Transaction::new_unsigned(message);
+        tx.signatures = vec![Signature::default(); actual_num_sigs as usize];
+        let _tx_bytes = serialize(&tx).unwrap();
```

Comment:
Remove this line?

Reply:
Nice! a525f96880a02ca1f937a5fbba73c673e0f22c26